project-base/storefront/cypress/docker/entrypoint.sh

```diff
@@ -7,7 +7,7 @@ if [ -n "$SPEC" ]; then
     echo "🎯 Running specific test: $SPEC"
     if [ "$TYPE" = "actual" ]; then
         echo "🔍 Running npm run specific-actual $SPEC"
-        npm run specific-actual "$SPEC"
+        npm run specific-regression "$SPEC"
```

Comment:
This seems to be a fault right? You are using $TYPE = "actual" but call regression?